### ISSUE TITLE
Images not showing during search

### ISSUE THREAD

**Issue by TylerJBraun** [JosefNemec/Playnite#3505]
### Bug Description

Images, like covers and backgrounds, show fine in the main screen and the media tab of the edit game details pane, but when I do a search for covers or backgrounds it shows a bunch of rectangles with the dimensions of the image under it. If I select one I can then see it in the media tab so the search is working, only it isn't showing the results in the Google image search pane.

### To Reproduce

It has been consistent and constant for many months. I don't know what triggered it. 

### Diagnostics ID

f0cd6ef0-1b6e-47c9-888c-d496c7f49f4b

### Screenshots

![image](https://github.com/JosefNemec/Playnite/assets/136659384/75922593-b6b4-4ae5-a0bd-3c76c40e0a5b)

**Comment by JosefNemec** [comment]
Can't reproduce, probably network issue or something.

![image](https://github.com/JosefNemec/Playnite/assets/3874087/9b779107-27c5-4fe0-92ca-36abbf99b7cc)

**Comment by TylerJBraun** [comment]
I don't think it's a network error as t has successfully fetched the
resolutions of the images and when I select one of the images that I can't
see it successfully loads it into the edit pane. I see now that I should
have included more screenshots. Sorry, I'll demonstrate with more now:
[image: image.png]
[image: image.png]
I did that exact search in browser and got this:
[image: image.png]
And sure enough, just as it shows in the browser, the third image in
Playnite's Google Image search matches the third image in Chrome's Google
Image search:
[image: image.png]
It fetches the images and tells me the resolutions of them, it just doesn't
display them until I select one. If I remember correctly it started right
after an update and I should have reported it right then so you'd know
which update it started with but I stupidly figured that someone else would
report it and it would get fixed in a future update which obviously never
happened. I've been picking each one with a vertical resolution and redoing
the search if I don't like it but have recently resorted (as seen above) to
duplicating the search in browser just to see the images but still choosing
it in Playnite.

If I'm the one who misunderstood you could you please explain further how a
network error could cause this all the time in Playnite's Google Image
search but affect nothing else? Our internet works fine for everything else
but I'm not a net tech (I'm barely a tech at all. I know just enough to
know how little I know) so I may just need more detailed help than that.

Thank you so much for the great program. I hope this is helpful.

**Comment by JosefNemec** [comment]
> It fetches the images and tells me the resolutions of them, it just doesn't
display them until I select one.

Sizes and items are fetched from the search web page itself (www.google.com), but image thumbnails are hosted on a different domain (encrypted-tbn0.gstatic.com). So my guess is that it's some connection issue to that domain which actually hosts thumbnail images. At least that's the only explanation I can think of.

You can try clearing Playnite's web cache (settings -> advanced) to see if it fixes the issue, but other than some external app blocking Playnite process from accessing that domain (like AV or Firewall), I can't think of anything else. Definitely not anything that could cause this from our side.

**Comment by JosefNemec** [comment]
Also try installing this: https://apps.microsoft.com/store/detail/webp-image-extensions/9PG2DK419DRG?hl=en-us&gl=us

It installs system wise support for webp images. Maybe Google is serving you results as webp images for some reasons.

**Comment by TylerJBraun** [comment]
Thanks for the additional info. That makes more sense. I had already tried
clearing the cache as per the 'stuff to do before posting a bug' post and
even though I don't have an aftermarket virus scanner or firewall I did go
into the Windows firewall and add ALL of Playnite's exes to the allowed
list and installed the webp app and restarted the computer but with no
success. I will take this info though and do some more research and get
back to you if I find anything that may be useful to you.

**Comment by JosefNemec** [comment]
Closing as this doesn't seem to be an issue on our side.

**Comment by Chrispako990210** [comment]
> Can't reproduce, probably network issue or something.
> 
> ![image](https://github.com/JosefNemec/Playnite/assets/3874087/9b779107-27c5-4fe0-92ca-36abbf99b7cc)

I can attest the initial issue does happen. Its inconsistent so hard to reproduce. Worked initially, i.e 5 min ago. Stopped fetching images all of a sudden, maybe after updating ish 10 games. Cleared the cache, restart and that work for another 5-6 images and now nothing works. Will try the webp extension and report back here.

Edit: Tried the webp and a system reboot, still not working. Hope it can get fixed, such a great feature...
@JosefNemec

**Comment by TylerJBraun** [comment]
Mine's consistently like that. Didn't know you could clear the cache. I
just hunted and found that in settings so I'm doing it now...
No luck. Still no images. Is there anything debuggy or diagnostic I can
send to help figure it out?

**Comment by JosefNemec** [comment]
https://api.playnite.link/docs/manual/gettingStarted/helpAndTroubleshooting/troubleshooting.html#web-image-search-doesnt-work

**Comment by TylerJBraun** [comment]
Thanks. I never saw that before. Unfortunately it didn't work. I installed
both the webp and av1 things and restarted Playnite and it didn't fix it.
I'll try restarting my computer too...
Nope. It's working fine on my other computer though, the newer one the
whole family uses.

**Comment by Luk3xx** [comment]
Just installed Playnite and I also can't see any images when doing web search.
Cache clear, WebP, AV1-AVIF - doesn't work.

[https://api.playnite.link/docs/manual/gettingStarted/helpAndTroubleshooting/troubleshooting.html#web-image-search-doesnt-work](url)
//If the issue persists after clearing the cache, then your network was blocked by Google from image search website (usually by anti-bot measures) preventing Playnite from getting search results. There is no solution for this other than to wait for block to be lifted or use something like VPN to appear under different IP to Google.//
 - This is pure copout pseudo-explanation. If Google blocked networks then no one would be able to pull any data, but it does retrieve data, just app doesn't display correctly images. I've tested multiple VPN's and locations, and it's the same, so clearly there is no "Google network block".

Update:
Tested app on few PC's, and it doesn't work on my main PC, but for example works on old laptop. Both have same windows 10, and some other things. Just different drivers.

Update 2:
It works in safe mode, but doesn't work in normal mode.
